src/app/reusableComponents/svg/NewsIcon/index.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+
+import './NewsIcon.scss';
+
+const NewsIcon = props => (
+    <svg
```

Comment:
If it is copied from somewhere I suggest to use a package instead

Reply:
Done.
I created new npm package for my svg icons written as React components.